firebase-config-interop/src/main/java/com/google/firebase/remoteconfig/interop/FirebaseRemoteConfigInterop.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.remoteconfig.interop;
+
+import androidx.annotation.NonNull;
+import com.google.firebase.remoteconfig.interop.rollouts.RolloutsStateSubscriber;
+
+public interface FirebaseRemoteConfigInterop {
+  void registerRolloutsStateSubscriber(
+      @NonNull String namespace, @NonNull RolloutsStateSubscriber subscriber);
```

Comment:
Can we add a comment for what the namespace is? Also maybe this could be a good place to explain how other SDKs should use this

Reply:
Yeah, good call.